readme.md

```diff
@@ -96,6 +96,11 @@ prompt pure
 
 ## Colors
 
+As explained in ZSH's[manual](http://zsh.sourceforge.net/Doc/Release/Zsh-Line-Editor.html#Character-Highlighting) color values can be:
+- a decimal integer up to the maximal number of color your terminal support
+- the name of one of the following nine colours: `black`, `red`, `green`, `yellow`, `blue`, `magenta`, `cyan`, `white` and `default` (the terminal’s default foreground)
+- ‘#’ followed by an RGB triplet in hexadecimal format (ex. `#424242`), *only* if your terminal support 24bit colors or when the module [`zsh/nearcolor`](http://zsh.sourceforge.net/Doc/Release/Zsh-Modules.html#The-zsh_002fnearcolor-Module) is loaded
```

Comment:
We should show an example of loading `zsh/nearcolor` and using a HEX color in the Example section. I think a lot of users will want this.

Reply:
Done.